experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricData.ts

```diff
@@ -56,14 +56,14 @@ export interface HistogramMetricData extends BaseMetricData {
  */
 export type MetricData = SingularMetricData | HistogramMetricData;
 
-export interface InstrumentationLibraryMetrics {
-  instrumentationLibrary: InstrumentationLibrary;
+export interface ScopeMetrics {
```

Comment:
Is `InstrumentationScopeMetrics` a name to be considered? Although it is somewhat verbose, we are not introducing any new term in the case.

Reply:
Oh, I found that this is the name in the OTLP proto.

Reply:
> Oh, I found that this is the name in the OTLP proto.

I considered `InstrumentationScopeMetrics` at first but, I think having this to as close as possible to the OTLP proto would make sense. :slightly_smiling_face:

I checked the other language SDKs but there does not seem to be a consensus on what to call it. Java uses this [`MetricData`](https://github.com/open-telemetry/opentelemetry-java/blob/a9968bcf323fab5fc793a33cab776910517f2249/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/MetricData.java) to send to the Exporter, .NET does not map the concept directly, and Python is currently in [the process of changing it to [`ScopeMetrics`](https://github.com/open-telemetry/opentelemetry-python/pull/2658). If that Python PR merges, we will have consistent naming across Python, and JS and proto. :slightly_smiling_face:

Reply:
I think its fine if we're matching the proto